app/build.gradle

```diff
@@ -1,14 +1,15 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
+apply plugin: "androidx.navigation.safeargs"
 
 android {
     compileSdkVersion 29
     buildToolsVersion "29.0.3"
 
     defaultConfig {
-        applicationId "cookpad.com.bottomnavwatson"
-        minSdkVersion 21
+        applicationId "com.cookpad.bottomnavwatson"
+        minSdkVersion 26
```

Comment:
Is this 26 instead of 21 due to the notification crashing? I think providing a PNG version of the notification icon would solve this.
On the other hand, it's not critical for a sample app.

Reply:
Nope, it's because of the channel creation for notifications, that forced to me to deal with fragmentation if I did not bump the version. So I just did it as this app it is just a showcase 👍